indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -256,7 +257,8 @@
 
       if (range && this.mark(range)) {
         // gutter uses this for positioning
-        this.anchorElement = this.marks[0];
+        // this.anchorElement = this.marks[0];
```

Comment:
```suggestion
```

Reply:
The anchor should be set to null if we can't determine a range (the else part of this if block).

Reply:
done